eunkyung/services/likeService.js

```diff
@@ -0,0 +1,13 @@
+const likeDao = require('../models/likeDao')
+
+
+
+const createLikePostByUserId = async (userId, postId) => {
```

Comment:
은경님 수정하시면서 해당 부분과 같이 여러줄이 띄어져 있는 부분도 수정해주세요!
코드를 구분하기 위해서는 한줄이 적당할 것 같습니다.
```suggestion
const likeDao = require('../models/likeDao')

const createLikePostByUserId = async (userId, postId) => {
```